frontend/src/components/doctors/doctors/DoctorSmallCard.tsx

```diff
@@ -17,21 +24,111 @@ function DoctorSmallCard({
 }: DoctorSmallCardProps) {
     const distance =
         locationForDistanceCalculation &&
-        getDoctorMinimalDistance(doctor, locationForDistanceCalculation, distanceUnit);
+        getDoctorMinimalDistance(
+            doctor,
+            locationForDistanceCalculation,
+            distanceUnit
+        );
+    const closestLocation =
+        locationForDistanceCalculation &&
+        getDoctorNearestLocation(doctor, locationForDistanceCalculation);
+
+    const image = doctor.image
+        ? doctor.image.toString()
+        : doctor.gender === "F"
+        ? "images/default-doctor-f.png"
+        : "images/default-doctor-m.png";
+
+    // TODO: replace with the real fields
+    const averageRating = 4.5;
+    const totalReviews = 10;
 
     return (
         <div className="doctorSmallCard" onClick={onClick}>
-            {doctor.fullName}
-            <br />
-            {doctor.category}
-            <br />
-            {doctor.specialities}
-            <br />
-            {distance && distance !== Infinity && (
-                <p>
-                    Distance: {distance.toFixed(2)} {distanceUnit}
-                </p>
-            )}
+            <img
+                className="doctorSmallCardImg"
+                src={image}
+                alt={doctor.fullName}
+            />
+            <div className="doctorSmallCardData">
+                <div>
+                    <h5>{doctor.fullName}</h5>
+                </div>
+                <div className="row w-100 m-0">
+                    <div className="bookmarkRibbon col-auto">
+                        <p className="text-black">
+                            {doctor.category || "No Category"}
+                        </p>
+                    </div>
+                    <div className="col">
+                        <span className="verification">
+                            {doctor.nancysNook && (
+                                <img
+                                    className="px-1"
+                                    src="images/nancynook.png"
+                                    alt="nancy nook"
+                                />
+                            )}
+                            {doctor.iCareBetter && (
+                                <OverlayTrigger
+                                    placement="bottom"
+                                    overlay={<Tooltip>View Profile</Tooltip>}
+                                >
+                                    <a
+                                        href={doctor.iCareBetter}
+                                        target="_blank"
+                                        rel="noreferrer"
+                                        onClick={(e) => e.stopPropagation()}
```

Comment:
I'm using chrome and e.stopPropagation() didn't solve the issue here. The card was opened first, and for some reason. this one got opened in the same tab :/